src/components/routes.js

```diff
@@ -1,25 +1,21 @@
-/* eslint-disable react/no-children-prop */
-import { RegisterForm } from './register-form'
-import { LoginForm } from './login-form'
-import { Main } from './main'
-import { NotFound } from './not-found'
-import React from 'react'
-import { Switch, Route } from 'react-router-dom'
+import {RegisterForm} from "./register-form";
+import {LoginForm} from "./login-form";
+import {Main} from "./main";
+import {About} from "./about";
+import React from "react";
+import {ResetPassword} from "./reset-password";
+import {Switch, Route} from "react-router-dom";
 
 export const Routes = () => (
-  <Switch>
-    <Route children={props => <LoginForm {...props} />} exact path="/" />
-    <Route
-      children={props => <RegisterForm {...props} />}
-      exact
-      path="/register"
-    />
-    <Route
-      children={props => <RegisterForm {...props} />}
-      exact
-      path="/register"
-    />
-    <Route children={props => <Main {...props} />} exact path="/main" />
-    <Route path="*"><NotFound /></Route>
-  </Switch>
-)
+    <Switch>
+        <Route children={props => <LoginForm {...props} />} exact path="/"/>
+        <Route children={props => <Main {...props} />} exact path="/main"/>
+        <Route children={props => <About {...props} />} exact path="/about"/>
+        <Route children={props => <ResetPassword {...props} />} exact path="/reset-password"/>
```

Comment:
Having the props in each line simplifies the reading / diffing on changes. Please change